### PR TITLE
chore: add Navigation in storybook (CALCOM-10760)

### PR DESCRIPTION
**Pull request by gitstart-calcom**
## Demo
https://www.loom.com/share/7ce9b388d5af471180f2a3f409997614

### CODE REVIEW

File: packages/ui/components/navigation/tabs/__stories__/horizontalTabs.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+<CustomArgsTable of={HorizontalTabs} />
+
+<Examples title="Default">
+  <Example title="Default">
+    <HorizontalTabs
+      tabs={[
+        {
+          name: "tab 1",
+          href: "http://localhost:6006/?path=/story/ui-navigation-horizontaltabs--horizontal-tabs/tab1",
+        },
```
Comment (Udit-takkar) [inline packages/ui/components/navigation/tabs/__stories__/horizontalTabs.stories.mdx:62]: same as https://github.com/calcom/cal.com/pull/10867/files#r1329125194

**Review by Udit-takkar — DISMISSED** [review]
Can we remove these from horizontal and vertical tabs?

Screenshot 2023-09-18 at 11 58 15 PM

**Comment by gitstart-calcom** [comment]
We made the changes, please take a look @Udit-takkar

**Comment by keithwillcode** [comment]
Screen Shot 2023-10-05 at 11 52 29 AM

Seeing this. Know why, @gitstart-calcom?

**Review by keithwillcode — DISMISSED** [review]
Screen Shot 2023-10-05 at 11 52 29 AM

Seeing this. Know why, @gitstart-calcom?

**Comment by gitstart-calcom** [comment]
@keithwillcode That's weird, we will look into that and provide a fix now

**Comment by gitstart-calcom** [comment]
@keithwillcode We noticed this is happening in all files, can we create a ticket to fix that?
image

**Review by PeerRich — APPROVED** [review]

**Pull request merged**